packages/ui/src/vue/components/ContentCode.vue

```diff
@@ -111,7 +191,15 @@ export default {
 
     toggleExpanded (key) {
       this.$set(this.expanded, key, !this.expanded[key])
+    },
+
+    localize (key, interpolations) {
```

Comment:
Why do we need to introduce this method (and the `HTMLhasLoaded` variable) instead of using a `v-if` (handling the absence or presence of `renderedHTML`) and the `localize` filter in the template?